Lib/Rabbit4000/tcpip/DMAETH100.LIB

```diff
@@ -869,38 +878,37 @@ ioi ld   (NBCR),a
    inc   hl
    ld    (ix+[_decb]+pd_rx_error),hl
 
-	ld		a,(ix+[_decb]+rcvlsbstat)
+   ; Again, where in the R6000 is this documented?
+#if !defined(_RAB6K)

```

Comment:
Oh no, I understand that.  I'm just trying to figure out the lack of documentation in the manual for those six bytes of status information.  The NBTESR register is just a curiosity for me.  What would that mean in a transmit scenario, and perhaps the inclusion of CRC and length check there is related to the lack of documentation regarding the entities I'm looking for in those other six status bytes.

And I hope you've figured out I've read the manual cover to cover a few times  ;-)